lib/note-window.js

```diff
@@ -0,0 +1,92 @@
+const electron = require('electron')
+const { app, remote } = electron
+const { BrowserWindow } = remote
+const path = require('path')
+const Config = require('electron-config')
+const config = new Config()
+const _ = require('lodash')
+
+const windowSize = config.get('windowsize') || {
+  x: null,
+  y: null,
+  width: 1080,
+  height: 720
+}
+
+function createNewWindow(currentNote) {
+  const mainWindow = new BrowserWindow({
+    x: windowSize.x,
+    y: windowSize.y,
+    width: windowSize.width,
+    height: windowSize.height,
+    useContentSize: true,
+    minWidth: 500,
+    minHeight: 320,
+    webPreferences: {
+      zoomFactor: 1.0,
+      enableBlinkFeatures: 'OverlayScrollbars'
+    },
+    icon: path.resolve(__dirname, '../resources/app.png')
+  })
+  const url = path.resolve(
+    __dirname,
+    process.env.NODE_ENV === 'development'
+      ? './main.development.html'
```

Comment:
This code will not works, I think because it been called in a different place so the `__dirname` is changed. Can you fix the path for this?